content/cudf-pandas/_index.html

```diff
@@ -138,7 +138,7 @@ <h2><i class="fa-light fa-bookmark"></i> Try Now on Colab</h2>
             <a href="https://nvda.ws/rapids-cudf" target="_blank"><img src="/images/OpenGoogleColab.png" class="img-w200"></a>
             <p>Take cuDF's new pandas accelerator mode for a test-drive in a free GPU-enabled notebook environment using your Google account by <a href="https://nvda.ws/rapids-cudf" target="_blank">launching on CoLab <i class="fa-solid fa-arrow-up-right"></i> </a></p>
             <br>
-            <p>cuDF's pandas accelerator mode works smoothly with all of the other RAPIDS libraries. Visit the RAPIDS <a href="https://rapids.ai/#quick-start" target="_blank">Quick Start</a> to get started with any RAPIDS library on your favorite platform.</p>
+            <p>cuDF's pandas accelerator mode is part of the cuDF package and works smoothly with all of the other RAPIDS libraries. Visit the RAPIDS <a href="https://rapids.ai/#quick-start" target="_blank">Quick Start</a> to get started with any RAPIDS library on your favorite platform.</p>
```

Comment:
And maybe tighten the sentence by simplifying "all of the other RAPIDS libraries" to "all RAPIDS libraries."
```suggestion
            <p>cuDF's pandas accelerator mode is part of the cuDF package and works smoothly with all RAPIDS libraries. Visit the RAPIDS <a href="https://rapids.ai/#quick-start" target="_blank">Quick Start</a> to get started with any RAPIDS library on your favorite platform.</p>
```